front/src/store.ts

```diff
@@ -7,7 +7,9 @@ import VueInstance from './main'
 
 Vue.use(Vuex)
 
+const API_BASE_PATH = '/api/v1'
 const INFO_URL = '/info'
+const DEFAULT_API_BASE_URL = 'http://localhost:8080'
```

Comment:
Maybe replace this by window.location.href with window.location.pathname ?

Default location will work with every url in production ?